plugins/gestures.koplugin/main.lua

```diff
@@ -134,7 +134,8 @@ function Gestures:isGestureAlwaysActive(ges, multiswipe_directions)
         end
     end
 
-    return self.gestures[ges] and self.gestures[ges].toggle_touch_input
+    return self.gestures[ges] and (self.gestures[ges].toggle_touch_input or
+        (self.gestures[ges].settings and self.gestures[ges].settings.always_active))
```

Comment:
You might want to add `touch_input_off` to the list, as people might expect it to always work, like toggle

Reply:
Sorry i meant `touch_onput_on`

Reply:
Yes, I got it, a good idea.

Reply:
Not sure how often this is called, but if on all gesture, to make it possibly less expensive, you could factor out the first table lookup:
```lua
local ges = self.gestures[ges] 
return ges and (ges.toggle_touch_input or ges.touch_input_on or...
```